android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -837,6 +838,107 @@ class GeckoEngine(
         )
     }
 
+    /**
+     * See [Engine.getTranslationsOfferPopup].
+     */
+    override fun getTranslationsOfferPopup(
+        onSuccess: (Boolean) -> Unit,
+        onError: (Throwable) -> Unit,
+    ) {
+        onSuccess(runtime.settings.translationsOfferPopup)
```

Comment:
If `translationsOfferPopup` is not asycn (returns righ way) why do we need `onSuccess` and `onError`? Maybe we should just change signature of the function to return `Boolean`?

Reply:
Thanks, I wasn't sure of the right pattern here, changed to just Boolean!